Tasks/Chyzh/Task_2/Task2_alt.py

```diff
@@ -0,0 +1,99 @@
+from datetime import datetime           # import class
+ch = int(input("Choose mode: 1 - for automatical output of current time; 2 - to enter current time: "))    #choose the proper mode
+if ch == 1:
+    curTime = str(datetime.now().time())    # get current time as a string
+    CT = list(curTime)[0:8]                 # turn string to list splitting milliseconds
+    print("".join(CT))                      # output of current time as a string
+elif ch == 2:
+    usersCurTime = input("Enter current time in hh:mm format:")     #input of current time
+    UCT = list(usersCurTime)
+    print(UCT)
+    hours = int(UCT[0] + UCT[1])
+    minutes = int(UCT[3] + UCT[4])
+    x = hours
+    y = minutes
```

Comment:
incorrect structure of the program, your main logic is placed inside this **else** clause, it will not be executed if user choses mode 1